app/Helpers/database/achievement.php

```diff
@@ -4,11 +4,6 @@
 use App\Platform\Enums\AchievementFlag;
 use App\Platform\Enums\AchievementPoints;
 use App\Platform\Enums\AchievementType;
-use App\Platform\Events\AchievementCreated;
-use App\Platform\Events\AchievementPointsChanged;
-use App\Platform\Events\AchievementPublished;
-use App\Platform\Events\AchievementTypeChanged;
-use App\Platform\Events\AchievementUnpublished;
```

Comment:
These events do not have to be dispatched explicitly anymore - moved to model events which guarantee consistent behaviour regardless of where models were created/changed: https://github.com/RetroAchievements/RAWeb/pull/2165/files#diff-af5d573c5bcfbd0ed76895c2569ebc277ce30f5b862d9d6e2cbe8b1f6102c014R111-R142

Reply:
I'll add accompanying tests for these changes

Reply:
added tests in https://github.com/RetroAchievements/RAWeb/pull/2165/commits/ebcb8c31b7b1ca6a1314591bc09bc17fc86845ae